### PR TITLE
flare-floss: init at 1.7.0

### PR DESCRIPTION
**Pull request by cole-h**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120001` at 462dc266 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>python38Packages.cxxfilt</li>
    <li>python39Packages.cxxfilt</li>
  </ul>
</details>
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>flare-floss</li>
    <li>python38Packages.plugnplay</li>
    <li>python39Packages.plugnplay</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/plugnplay/default.nix:15:0:
```
   |
15 |     description = "A Generic plug-in system for python applications";
   | ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/tools/security/flare-floss/default.nix:27:0:
```
   |
27 |     description = "Automatically extract obfuscated strings from malware";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/cxxfilt/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/cxxfilt/default.nix:6]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/cxxfilt/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  pname = "cxxfilt";
+  version = "0.2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "ef6810e76d16c95c11b96371e2d8eefd1d270ec03f9bcd07590e8dcc2c69e92b";
+  };
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/cxxfilt/default.nix:14]: missing checkPhase and pythonImportsCheck.

File: pkgs/development/python-modules/plugnplay/default.nix [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/plugnplay/default.nix:5]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/plugnplay/default.nix [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  pname = "plugnplay";
+  version = "0.5.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "877e2d2500a45aaf31e5175f9f46182088d3e2d64c1c6b9ff6c778ae0ee594c8";
+  };
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/plugnplay/default.nix:13]: missing checkPhase and pythonImportsCheck.

File: pkgs/development/python-modules/viv-utils/default.nix [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+, buildPythonPackage
+, isPy3k
+, fetchFromGitHub
+, funcy
+, pefile
+, vivisect
+, intervaltree
+, setuptools
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/viv-utils/default.nix:11]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/viv-utils/default.nix [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  '';
+
+  propagatedBuildInputs = [
+    funcy
+    pefile
+    vivisect
+    intervaltree
+    setuptools
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/viv-utils/default.nix:35]: missing checkPhase and pythonImportsCheck.

File: pkgs/development/python-modules/vivisect/default.nix [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+, buildPythonPackage
+, isPy3k
+, fetchPypi
+, pyasn1
+, pyasn1-modules
+, cxxfilt
+, msgpack
+, pycparser
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/vivisect/default.nix:11]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/vivisect/default.nix [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+    msgpack
+    pycparser
+  ];
+
+  preBuild = ''
+    sed "s@==.*'@'@" -i setup.py
+  '';
+
+  # requires another repo for test files
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/vivisect/default.nix:34]:
```suggestion
  doCheck = false;
  
  pythonImportsCheck = [ "vivisect" ];
```

File: pkgs/tools/security/flare-floss/default.nix [hunk 9]
```diff
@@ -0,0 +1,32 @@
+{ lib
+, python2
+, fetchFromGitHub
+}:
+python2.pkgs.buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/tools/security/flare-floss/default.nix:5]:
```suggestion
}:

python2.pkgs.buildPythonPackage rec {
```

File: pkgs/tools/security/flare-floss/default.nix [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+  propagatedBuildInputs = with python2.pkgs; [
+    pyyaml
+    simplejson
+    tabulate
+    vivisect
+    plugnplay
+    viv-utils
+    enum34
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/tools/security/flare-floss/default.nix:25]: missing checkPhase and pythonImportsCheck.

**Pull request merged**